### PR TITLE
[VCDA-4459] Fix node pool section in schema.json 

### PR DESCRIPTION
**Pull request by Anirudh9794**
Signed-off-by: Aniruddha Shamasundar <aniruddha.9794@gmail.com>

## Description
Please provide a brief description of the changes proposed in this Pull Request

- Update node pool section in the schema to proper format

## Checklist
- [x] tested locally
- [ ] updated any relevant dependencies
- [ ] updated any relevant documentation or examples

## API Changes
Are there API changes?
- [ ] Yes
- [x] No

If yes, please fill in the below

1. Updated conversions?
    - [ ] Yes
    - [ ] No
    - [x] N/A
2. Updated CRDs?
    - [ ] Yes
    - [ ] No
    - [x] N/A
3. Updated infrastructure-components.yaml?
    - [ ] Yes
    - [ ] No
    - [x] N/A
4. Updated `./examples/capi-quickstart.yaml`?
    - [ ] Yes
    - [ ] No
    - [x] N/A
5. Updated necessary files under `./infrastructure-vcd/v1.0.0/`?
   - [ ] Yes
   - [ ] No
   - [x] N/A

## Issue
If applicable, please reference the relevant issue

Fixes #

- - -
This change is [Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/268)

### CODE REVIEW

**Review by lzichong — APPROVED** [review]
:lgtm:, will update docs after merge :)

> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/268#-:-NCaqLArBRDBKUms1_tR:bkthhpu)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @Anirudh9794)

**Review by sahithi — APPROVED** [review]
:lgtm: 

> *[Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/268#-:-NCb3R0NEATx8mvwWN_I:bg7igkv)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @Anirudh9794)

**Pull request merged**
